python/BioSimSpace/FreeEnergy/_relative.py

```diff
@@ -696,6 +1278,159 @@ def _difference(self, pmf_ref):
         # Now call the staticmethod passing in both PMFs.
         return Relative.difference(pmf, pmf_ref)
 
+    @staticmethod
+    def check_overlap(overlap, estimator="MBAR"):
```

Comment:
For agreement with the current API naming conventions, this should be `checkOverlap`.